examples/task.hpp

```diff
@@ -268,6 +268,12 @@ class basic_task {
     __coro::coroutine_handle<_promise> coro_;
     std::optional<awaiter_context_t<_promise, ParentPromise>> context_{};
 
+    ~_task_awaitable() {
+        if (_coro) {
+            _coro.destroy();
```

Comment:
Thanks a lot for the fix. Here the member is named `coro_`, so we need to change that to
 ```suggestion
        if (coro_) {
            coro_.destroy();
```

Reply:
> Thanks a lot for the fix. Here the member is named `coro_`, so we need to change that to

Sorry.  That's what I get for having an epiphany at 1am and then, when it solved the issue, excitedly trying to transpose it to this task implementation.